system/TestAuditSuite.py

```diff
@@ -0,0 +1,184 @@
+import pytest
+from system.utils import *
+
+
+@pytest.mark.usefixtures('docker_setup_and_teardown')
+class TestAuditSuite:
+
+    @pytest.mark.asyncio
+    async def test_case_restart_one_node(self, pool_handler, wallet_handler, get_default_trustee):
+        trustee_did, _ = get_default_trustee
+        hosts = [testinfra.get_host('ssh://node{}'.format(i)) for i in range(1, 8)]
+        for i in range(15):
+            await nym_helper(pool_handler, wallet_handler, trustee_did, random_did_and_json()[0], None, None, None)
+        output = hosts[5].check_output('systemctl restart indy-node')
+        print(output)
+        for i in range(30):
+            await nym_helper(pool_handler, wallet_handler, trustee_did, random_did_and_json()[0], None, None, None)
```

Comment:
I had a hard time trying to understand what this code does until I found that this is actually just sends NYM transaction (and there is actually whole bunch of similar functions for other transaction types). I think it would be much better to rename such functions to something like `send_nym` to make intent much more clear and test more readable.

Reply:
Done.